src/commands/login/index.ts

```diff
@@ -92,6 +92,7 @@ fastify.get("/complete", async (request: any, reply: any) => {
   console.log('You have successfully authenticated with the server.')
   console.log('')
   fastify.close()
+  process.exit(1)
```

Comment:
Wondering should this not be `0` since a non-zero exit code implies there was an error.
In this case, getting to this point implies a successful flow; hence 0 should be the process exit code?
```suggestion
  process.exit(0)
```